packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -70,6 +87,12 @@ export class ApplicationLoadBalancedFargateService extends ApplicationLoadBalanc
     } else if (props.taskDefinition) {
       this.taskDefinition = props.taskDefinition;
     } else if (props.taskImageOptions) {
+      this.validateHealthyPercentage('containerCpu', props.containerCpu);
+      this.validateHealthyPercentage('containerMemoryLimitMiB', props.containerMemoryLimitMiB);
```

Comment:
The method `validateHealthyPercentage` is for healthy percentage, so I don't think it is good to use it here. Actually the method can be used here because the validation process is same, but could you add another methods for each cpu and memory?

Reply:
Thank you for pointing this out. We will correct it.